downscale/read_config.f90

```diff
@@ -40,6 +40,9 @@ subroutine read_config (fname, n, config_names, values)
   config_names(18) = "STN_END_DATE"   !  
   config_names(19) = "GEN_STA_WEIGHTS"   !  
   config_names(20) = "STA_WEIGHT_NAME"   !  
+  config_names(21) = "NPREDICT"          !total number of predictors (6+number of NWP predictors)
```

Comment:
can we calculate this on the fly by taking length of nwp vars (+6); we may want to make the list of 'stat vars' also configurable.